src/pretix/control/views/vouchers.py

```diff
@@ -240,8 +244,13 @@ def get_form_kwargs(self):
     @transaction.atomic
     def form_valid(self, form):
         form.instance.event = self.request.event
-        messages.success(self.request, _('The new voucher has been created: {code}').format(code=form.instance.code))
         ret = super().form_valid(form)
+        url = reverse('control:event.voucher', kwargs={
+            'organizer': self.request.event.organizer.slug,
+            'event': self.request.event.slug,
+            'voucher': self.object.pk
+        })
+        messages.success(self.request, _format_html_lazy(_('The new voucher has been created: <a href="{url}">{code}</a>'), code=form.instance.code, url=url))
```

Comment:
Translators tend to screw up HTML markup. Could we leave the translated string as-is and instead pass in the full link markup for ``{code}``?